src/utils/progressbar.js

```diff
@@ -0,0 +1,64 @@
+import React, {useState} from 'react';
+import {View, StyleSheet, Dimensions, Button, Text} from 'react-native';
+
+import ProgressBarAnimated from 'react-native-progress-bar-animated';
+
+const ProgressBar = () => {
+  const [progress, setProgress] = useState(70);
+
+  const barWidth = Dimensions.get('screen').width - 30;
+
+  return (
+    <View style={styles.container}>
+      <View>
```

Comment:
why do you need this view?

Reply:
This was temporarily to hold the progressbar and buttons, which I should've mentioend they were for demo purposes.